server/osc_dash.py

```
@@ -133,149 +213,372 @@ def api_backtest(
+    if today_file.exists():
+        today_ticks = _count_lines_fast(today_file)
+    return {
+        "running": running,
+        "pid": _collector_proc.pid if running else None,
+        "total_ticks_collected": today_ticks,
+    }
+
+
+@app.post("/api/collector/start")
+def api_collector_start():
+    global _collector_proc
```

Ruff (0.16.2)

[warning] Using the global statement to update `_collector_proc` is discouraged

(PLW0603)